syscall.go

```diff
@@ -10,6 +10,13 @@ import "unsafe"
 
 const maxArgs = 9
 
+// SyscallN takes fn, a C function pointer and a list of arguments as uintptr.
+// There is an internal maximum number of arguments that SyscallN can take. It panics
+// when the maximum is exceeded. It returns the result and the libc error code if there is one.
+//
+// NOTE: SyscallN does not properly call functions that have both integer and float parameters.
```

Comment:
This issue is only for Arm, right?

Reply:
> I looked it up for amd64 and according the the [System V ABI](https://refspecs.linuxbase.org/elf/x86_64-abi-0.99.pdf) which macOS and linux use: "if the class is SSE, the next available vector register of the sequence %xmm0, %xmm1 is used."
It affects amd64 in macOS, linux and other System V compliant OSs.

Reply:
Acknowledged. This seems different from Windows.